cli/cmd/auth.go

```diff
@@ -101,6 +101,7 @@ More precisely, the Keptn CLI stores the endpoint and API token using *pass* in
 			}
 
 			logging.PrintLog("Successfully authenticated", logging.InfoLevel)
+			logging.PrintLog("CLI is authenticated against the Keptn cluster "+*authParams.endPoint, logging.InfoLevel)
```

Comment:
Thanks for your contribution!
Can we please combine those messages
`logging.PrintLog("Successfully authenticated against the Keptn cluster "+*authParams.endPoint, logging.InfoLevel)`

Reply:
@agrimmer Thanks for reviewing, have combined the messages :)